.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2022 Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# SPDX-License-Identifier: curl
+
+name: spell
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - '**.md'
+    - '!docs/BINDINGS.md'
```

Comment:
I think paths are excluded/ignored by specifying them in another key called `paths-ignore` instead of `paths`.